CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 45.0.0
```

Comment:
Yeah normally there's one PR to do the changes, then a separate PR to bump the version and release it. This ensures two competing PRs with the same version number but different changes don't get released.